Makefile

```diff
@@ -147,28 +148,40 @@ pull-docs:
 
 generate.init: $(TERRAFORM_PROVIDER_SCHEMA) pull-docs
 
-# ====================================================================================
-# Test utilities
-# TODO(muvaf): Move most of this to build submodule.
-
-uptest: $(KIND) $(KUBECTL) $(HELM3) $(UP) $(KUTTL)
-	@$(INFO) running uptest using kind $(KIND_VERSION)
-	@./cluster/install_provider.sh || $(FAIL)
-	@echo "$${UPTEST_EXAMPLE_VALUE_REPLACEMENTS}" > $(WORK_DIR)/replacements.yaml
-	@KIND=$(KIND) KUBECTL=$(KUBECTL) KUTTL=$(KUTTL) go run github.com/upbound/official-providers/testing/cmd --data-source "$(WORK_DIR)/replacements.yaml" || $(FAIL)
-
-uptest-local: $(KUBECTL) $(KUTTL)
-	@$(INFO) running automated tests with uptest using current kubeconfig $(KIND_VERSION)
-	@KUBECTL=$(KUBECTL) KUTTL=$(KUTTL) go run github.com/upbound/official-providers/testing/cmd || $(FAIL)
-
-cluster_dump: $(KUBECTL)
-	@mkdir -p ${DUMP_DIRECTORY}
-	@$(KUBECTL) cluster-info dump --output-directory ${DUMP_DIRECTORY} --all-namespaces || true
-	@$(KUBECTL) get managed -o yaml > ${DUMP_DIRECTORY}/managed.yaml || true
-	@cat /tmp/automated-tests/case/*.yaml > ${DUMP_DIRECTORY}/kuttl-inputs.yaml
-
 .PHONY: pull-docs
 
+# ====================================================================================
+# End to End Testing
+CROSSPLANE_NAMESPACE = upbound-system
+-include build/makelib/local.xpkg.mk
+-include build/makelib/controlplane.mk
+
+# This target requires the following environment variables to be set:
+# - UPTEST_EXAMPLE_LIST, a comma-separated list of examples to test
+# - UPTEST_CLOUD_CREDENTIALS (optional), cloud credentials for the provider being tested, e.g. export UPTEST_CLOUD_CREDENTIALS=$(cat ~/.aws/credentials)
```

Comment:
This line mentions aws-related stuff. It would be great to include an example that works for the GCP provider.